src/Rings/mpolyquo-localizations.jl

```diff
@@ -1055,6 +1055,13 @@ function ==(f::MPolyQuoLocalizedRingHom, g::MPolyQuoLocalizedRingHom)
   return true
 end
 
+function Base.hash(f::MPolyQuoLocalizedRingHom, h::UInt)
+  b = 0xd6d389598ad28724  % UInt
+  h = hash(domain(f), h)
+  h = hash(codomain(f), h)
```

Comment:
Same here.

Reply:
Done in 98c94979a6d4e4ae204aae9f6d5f695278e70fd6